### PR TITLE
{chem}[foss/2019a] Spex v05.00, FLEUR v0.30-MaXR3.1, Wannier90 v3.0.0 (WIP)

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**FAILED**
Build succeeded for 4 out of 6 (6 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/b4c13343f4cd2cdba4eba0c90fd6acfd for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/ced3e9e53ec29bc7f56493eb12b85178 for a full test report.

File: easybuild/easyconfigs/f/FLEUR/FLEUR-0.30-MaXR3.1-foss-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+easyblock = 'CMakeMake'
+
+name = 'FLEUR'
+version = '0.30-MaXR3.1'
+
+homepage = 'http://www.flapw.de/'
```
Comment (Micket) [inline easybuild/easyconfigs/f/FLEUR/FLEUR-0.30-MaXR3.1-foss-2019a.eb]: https

File: easybuild/easyconfigs/f/FLEUR/FLEUR-0.30-MaXR3.1-foss-2019a.eb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+description = """FLEUR is a feature-full, freely available FLAPW (full potential linearized augmented planewave) code,
+ based on density-functional theory."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'usempi': True, 'r8': True}
+
+source_urls = ['http://www.flapw.de/pm/uploads/FLEUR/']
```
Comment (Micket) [inline easybuild/easyconfigs/f/FLEUR/FLEUR-0.30-MaXR3.1-foss-2019a.eb]: https

File: easybuild/easyconfigs/f/FLEUR/FLEUR-0.30-MaXR3.1-intel-2019a.eb [hunk 3]
```diff
@@ -0,0 +1,42 @@
+easyblock = 'CMakeMake'
+
+name = 'FLEUR'
+version = '0.30-MaXR3.1'
+
+homepage = 'http://www.flapw.de/'
```
Comment (Micket) [inline easybuild/easyconfigs/f/FLEUR/FLEUR-0.30-MaXR3.1-intel-2019a.eb]: https

File: easybuild/easyconfigs/f/FLEUR/FLEUR-0.30-MaXR3.1-intel-2019a.eb [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+description = """FLEUR is a feature-full, freely available FLAPW (full potential linearized augmented planewave) code,
+ based on density-functional theory."""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
+toolchainopts = {'usempi': True, 'r8': True}
+
+source_urls = ['http://www.flapw.de/pm/uploads/FLEUR/']
```
Comment (Micket) [inline easybuild/easyconfigs/f/FLEUR/FLEUR-0.30-MaXR3.1-intel-2019a.eb]: https

File: easybuild/easyconfigs/s/Spex/Spex-05.00-foss-2019a.eb [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+homepage = 'https://spex.readthedocs.io'
+description = """Spex is a computer code based on many-body perturbation theory.
+ It uses the all-electron full-potential linearized augmented plane-wave method (FLAPW),
+ which provides an accurate basis set for all kinds of materials including transition metals,
+ oxides, and even f-electron systems."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['http://iffwww.iff.kfa-juelich.de/~cfried/archive/']
```
Comment (Micket) [inline easybuild/easyconfigs/s/Spex/Spex-05.00-foss-2019a.eb]: https

File: easybuild/easyconfigs/s/Spex/Spex-05.00-intel-2019a.eb [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+homepage = 'https://spex.readthedocs.io'
+description = """Spex is a computer code based on many-body perturbation theory.
+ It uses the all-electron full-potential linearized augmented plane-wave method (FLAPW),
+ which provides an accurate basis set for all kinds of materials including transition metals,
+ oxides, and even f-electron systems."""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
+
+source_urls = ['http://iffwww.iff.kfa-juelich.de/~cfried/archive/']
```
Comment (Micket) [inline easybuild/easyconfigs/s/Spex/Spex-05.00-intel-2019a.eb]: https

File: easybuild/easyconfigs/s/Spex/Spex-05.00_missing_type.patch [hunk 7]
```diff
@@ -0,0 +1,12 @@
+declare type of loop index, otherwise some compilers complain
```
Comment (boegel) [inline easybuild/easyconfigs/s/Spex/Spex-05.00_missing_type.patch]: @migueldiascosta Missing author or upstream reference?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 6 (6 easyconfigs in this PR)
node3116.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/e7518684b25979055aa28576f77160a5 for a full test report.

**Comment by migueldiascosta** [comment]
hm, `spex` package changed in place. Also documentation says "the Spex source code cannot be downloaded directly". Leaving this WIP for now.

**Comment by boegel** [comment]
No changes for over 2 years, still marked as WIP, so I'll go ahead and close this PR...

@migueldiascosta If this is still relevant, please open a new PR (perhaps with a more recent toolchain)

**Pull request closed without merging**